<?xml version="1.0"?>
<launch>
    <node name="safety_node" pkg="cbot_monitoring" type="safety_node" respawn="true" output="screen">
    </node>
</launch>
